extensions/amp-viqeo-player/validator-amp-viqeo-player.protoascii

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-viqeo-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-viqeo-player"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    deprecated_allow_duplicates: true
```

Comment:
Please remove this line. This is only for old extensions which are grandfathered due to the damage changing them would cause.